plugins/google-cloud/tests/datastore.spec.ts

```diff
@@ -120,14 +120,14 @@ describe('Google Cloud Storage', () => {
           return {
             __esModule: true,
             default: class Foo extends originalModule {
-              datastore: any;
-              constructor(props) {
+              public datastore: object;
```

Comment:
Did you try here `any`? ....

Reply:
Ahh, on test is fine use `object`.

Reply:
As is a test, we can bypass any with object